src/components/layout/seo.tsx

```diff
@@ -102,9 +102,8 @@ const SEO = ({
    * The i18n object (languages, language, originalPath, defaultLanguage)
    * is replaced with constants and locales data for building the alternate meta tag.
    */
-  const dataNode = locales.edges.find(
-    (e) => e.node.ns === 'translations',
-  )?.node;
+  const dataNode = locales.edges.find((e) => e.node.ns === 'translations')
+    ?.node;
```

Comment:
This formatting change leads to the failing test. `yarn format` will revert it to the correct format.

Reply:
`yarn format` did not revert it. I did it manually but after I ran `yarn format` again it formatted it to the form that causes the test failure

Reply:
It worked after running `yarn install --frozen-lockfile`